uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/services/document.js

```diff
@@ -312,7 +312,7 @@ define([
           contactIds.push(document.source_contact_id);
 
           if (document.assignee_contact_id && document.assignee_contact_id.length) {
-            contactIds.push(document.assignee_contact_id[0]);
+            Array.prototype.push.apply(contactIds, document.assignee_contact_id);
```

Comment:
Cant we can simply push using `contactIds.push(document.assignee_contact_id)`?

Reply:
yes we can. Will fix it.

Reply:
Can we use `contactIds = contactIds.concat(document.assignee_contact_id)`? which is more readable in my opinion.

Reply:
sounds more appropriate, will do.